### PR TITLE
Instrument start time

### PR DESCRIPTION
**Pull request by ylecuyer**
Instrument start time for workers/launcher and expose them through Puma.stats

### CODE REVIEW

File: lib/puma/cluster.rb [hunk 1]
```diff
@@ -95,12 +95,13 @@ def initialize(idx, pid, phase, options)
         @signal = "TERM"
         @options = options
         @first_term_sent = nil
+        @started_at = Time.now
```
Comment (nateberkopec) [inline lib/puma/cluster.rb]: cluster inherits from runner, so I believe this is already initialized, no?

**Comment by nateberkopec** [comment]
Code looks fine, but what's the use case here?

Isn't this info already available in `ps aux`, for example?

**Comment by nateberkopec** [comment]
Also wondering how this interacts with restarts and child process kills/restarts

**Comment by ylecuyer** [comment]
> Code looks fine, but what's the use case here?
> 
> Isn't this info already available in `ps aux`, for example?

This is the use case https://github.com/ylecuyer/puma-status

> Also wondering how this interacts with restarts and child process kills/restarts

When I tested, the `started_at` field is reset at restart

File: lib/puma/cluster.rb (hunk 1 above)
Comment (ylecuyer) [reply lib/puma/cluster.rb]: But this is for the `Worker` class as you can see L89

**Comment by SuperQ** [comment]
Nice, thanks, this will also be useful for our Prometheus metrics endpoint.

**Pull request merged**